public/docs/dart/latest/guide/_data.json

```diff
@@ -5,6 +5,10 @@
     "title": "Step By Step Guide"
   },
 
+  "cheatsheet": {
+    "title": "Angular Cheat Sheet"
+  },
+
```

Comment:
This adds the cheatsheet to the navigation under the Dart bit of the site - you might not want to do this yet